infrastructure/main.tf

```diff
@@ -51,6 +51,10 @@ module "wa_workflow_api_database_flex" {
   location                    = var.location
   business_area               = "cft"
   env                         = var.env
+  action_group_name           = join("-", [local.db_name, var.action_group_name])
+  email_address_key           = var.email_address_key
+  email_address_key_vault_id  = data.azurerm_key_vault.wa_key_vault.id
+
```

Comment:
you need to set cpu_threshold, memory_threshold and storage_threshold here 
otherwise it will use default value of 80%

Reply:
cpu_threshold = var.cpu_threshold
  memory_threshold = var.memory_threshold
  storage_threshold = var.storage_threshold

Reply:
yep thats why it didn't work